src/main/java/io/mosip/digitalcard/controller/DigitalCardController.java

```diff
@@ -85,7 +85,7 @@ public ResponseEntity<?> credentialEvent(@RequestBody EventModel eventModel)  {
             additionalAttributes.remove("credential");
             additionalAttributes.remove("protectionKey");
             additionalAttributes.remove("proof");
-            digitalCardServiceImpl.generateDigitalCard(eventModel.getEvent().getData().get("credential").toString(),
+            digitalCardServiceImpl.generateDigitalCard(eventModel.getEvent().getData().containsKey("credential")?eventModel.getEvent().getData().get("credential").toString():null,
```

Comment:
We have tested this ? I believe key of credential can be there, but value might be null.